api/graphql/makeModels.js

```diff
@@ -111,6 +111,7 @@ export default async function makeModels (userId, isAdmin) {
         'memberships',
         'moderatedCommunityMemberships',
         'locationObject',
+        {events: {querySet: true}},
```

Comment:
i think we should probably call this eventsAttending

Reply:
I agree and would want this reflected in the model too, if that is in fact what is being queried. I'm not actually clear. Is the `User#events` relation only pulling events to which I'm attending or also events I've merely created? 

If I remember correctly I'm always added to the event group ("attending") when I create an event. If that is the case I think the query and this naming is correct.

Reply:
@lorenjohnson That's correct, the event creator is also added to the event group, and therefore "attending"

Reply:
@wushenshen so lets call it eventsAttending then, right? I think if a creator of an event says they are no longer attending the event it should disappear from the list on their profile